views/webhooks-attemptsRows.html

```diff
@@ -1,10 +1,10 @@
 {{#each attempts}}
 	<tr class="{{#if error}}error{{/if}}">
-		<td>{{date}}</td>
+		<td data-sort-value="{{date}}">{{date}}</td>
 		<td>{{time}}</td>
 		<td>{{sent}}</td>
 		<td>{{received}}</td>
 		<td>{{retriesLeft}}</td>
```

Comment:
Since `retiresLeft` is of type number, you need to either specify the sorted value directly in the cell:
```suggestion
		<td data-sort-value="{{retiresLeft}}">{{retriesLeft}}</td>
```
or define it at the column header, like @guillegr123 suggested.